src/shares/PaymentHub.sol

```diff
@@ -57,16 +57,17 @@ contract PaymentHub {
         weth = uniswapQuoter.WETH9();
         priceFeedCHFUSD = AggregatorV3Interface(_aggregatorCHFUSD);
         priceFeedETHUSD = AggregatorV3Interface(_aggregatorETHUSD);
-
     }
 
-    function getPriceInEther(uint256 amountOfXCHF) external returns (uint256) {
-        return getPriceInEther(amountOfXCHF, address(0));
-    }
+    // I would remove that function. It is equivalent to calling getPriceInEtherFromOracle, but more costly.
+    //function getPriceInEther(uint256 amountOfXCHF) external returns (uint256) {
+      //  return getPriceInEther(amountOfXCHF, address(0));
+    //}
```

Comment:
Yes, but removing this would break backwards compatibility. A simultaneous update of all Brokerbot PaymentHub's and also the Brokerbot widgets for all clients would be necessary. Right?
The widget currently calls this to get the exchange rate.

Reply:
Good point, added it again